packages/fx-core/resource/package.nls.json

```diff
@@ -272,6 +272,10 @@
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate.mitigation": " Mitigation: create Adaptive Card template in \"%s\" and then update \"%s\" field to the relative path in \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly": "There is no required parameter defined in API \"%s\". The first optional parameter is set as the parameter for command \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly.mitigation": " Mitigation: If \"%s\" is not what you need, edit the parameter of command \"%s\" in \"%s\". The parameter name should match what's defined in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.pluginManifest.missingFunctionDescription": "Missing description of function \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.pluginManifest.missingFunctionDescription.mitigation": " Mitigation: Update description of \"%s\" in \"%s\"",
+  "core.copilotPlugin.scaffold.summary.warning.pluginManifest.functionDescription.lengthExceeding": "Description of function \"%s\" has been truncated to %s characters to meet the length requirement.",
```

Comment:
Proposed: Description of function \"%s\" shortened to %s characters to meet the length requirement."